ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -121,12 +174,48 @@ export const CoinOverview = ({
     if (isEvm) {
       if (showNativeTokenAsMainBalance) {
         return (
-          <PercentageAndAmountChange
-            value={tokensMarketData?.[zeroAddress()]?.pricePercentChange1d}
-          />
+          <Box className="wallet-overview__currency-wrapper">
+            <PercentageAndAmountChange
+              value={tokensMarketData?.[zeroAddress()]?.pricePercentChange1d}
+            />
+            <ButtonLink
+              endIconName={IconName.Export}
+              endIconProps={{
+                size: IconSize.Sm,
+                color: IconColor.primaryDefault,
+                className: 'wallet-overview__portfolio_button',
+              }}
+              size={ButtonLinkSize.Inherit}
+              onClick={handlePortfolioOnClick}
+              as="a"
+              data-testid="portfolio-link"
+              className="wallet-overview__portfolio_text"
+            >
+              {t('portfolio')}
+            </ButtonLink>
+          </Box>
         );
       }
-      return <AggregatedPercentageOverview />;
+      return (
+        <Box className="wallet-overview__currency-wrapper">
+          <AggregatedPercentageOverview />
+          <ButtonLink
+            endIconName={IconName.Export}
+            endIconProps={{
+              size: IconSize.Sm,
+              color: IconColor.primaryDefault,
+              className: 'wallet-overview__portfolio_button',
+            }}
+            size={ButtonLinkSize.Inherit}
+            onClick={handlePortfolioOnClick}
+            as="a"
+            data-testid="portfolio-link"
+            className="wallet-overview__portfolio_text"
```

Comment:
The goal should be to use design system components out of the box without making modifications. If the designs aren't align, then maybe we should take a look and discuss with designer on why they chose to customize further. 

As for the `TextVariant`, I was able to get it working without the need of the className. 

```suggestion
            textProps={{ variant: TextVariant.bodyMdMedium }}
```

Reply:
Thanks @garrettbear ! Fixed here [52a8669](https://github.com/MetaMask/metamask-extension/pull/27161/commits/52a8669072e05aef09347df60e9ae1459dd543d3)